adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Protocol.java

```diff
@@ -0,0 +1,18 @@
+package org.jvalue.ods.adapterservice.adapter;
+
+import org.jvalue.ods.adapterservice.adapter.importer.HttpImporter;
+import org.jvalue.ods.adapterservice.adapter.importer.Importer;
+import org.springframework.web.client.RestTemplate;
+
+public enum Protocol {
```

Comment:
Will it work if someone uses lower case `http` or `Http` in the requesting REST API call?

Reply:
> > Can `Protocol` and `Format` be in lower case `http` or `json` in a REST request?
> 
> No, but it has not worked with lower case before, so nothing has changed from the endpoint user perspective.
> 
> If we want to support lower case `http` or even `Http` then we can do that by using a custom Jackson deserializer. But that should be done in a separate PR.

Couldn't we just extend the existing enum by the other spellings? I agree, however, that we do not need this feature and should leave it out anyway

Reply:
Yes, technically that works, too. But I would consider having enums with different spellings as very bad design.